js/repositories.bzl

```diff
@@ -44,3 +47,11 @@ def rules_js_dependencies():
         #url = "https://github.com/aspect-build/bazel-lib/archive/refs/tags/v{}.tar.gz".format(versions.bazel_lib),
         url = "https://github.com/aspect-build/bazel-lib/archive/{}.tar.gz".format(versions.bazel_lib),
     )
+
+    maybe(
+        http_archive,
+        name = "typescript",
+        build_file = "//ts:BUILD.typescript",
+        sha256 = "70d5d30a8ee92004e529c41fc05d5c7993f7a4ddea33b4c0909896936230964d",
+        urls = ["https://registry.npmjs.org/typescript/-/typescript-4.6.3.tgz"],
```

Comment:
This means the version is locked with the version of rules_js?

Reply:
No, take a look at the commit description, already preemptive answered that 😜

Reply:
Genius 🤦‍♂️